### DISCUSSION TITLE
Mobi imported to PKSIM for population simulation

### DISCUSSION THREAD

**Discussion by hainiwen** [Open-Systems-Pharmacology/Forum#1755] [Questions & Problems]
Hi people, i learned that we should be able to export Mobi simulations to PKSIM and run population simulation from there. 
My model is a self define model, and when I import it to PKSIM and try to add variabilities to parameters that I am interested in, I cant find many parameters there. 

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/56790121/09b29e51-0d9f-4b5a-8e03-4478030bb63b)

Does anyone know where I went wrong? How can I get around this?

Regards,
Haini

**Comment by PavelBal** [comment]
Make sure that you enable the "Can be varied in population" checkbox for the parameter in MoBi:

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25903040/0ae3a024-9d6e-48ad-af38-f43205544d75)
